src/main/java/com/assignment/Assignment5.java

```diff
@@ -0,0 +1,10 @@
+package com.assignment;
+
+import java.util.Arrays;
+
+public class Assignment5 {
+
+    public int returnPlayerIndex(String [] Players,String indexString) {
```

Comment:
The naming convention for a method should be :
get -> if its returning something
set -> if its setting something